tests/test_sign_cmd.py

```diff
@@ -80,6 +80,12 @@ def test_sign_tx(backend, firmware, navigator, test_name):
                                                       "Hold to sign",
                                                       ROOT_SCREENSHOT_PATH,
                                                       test_name)
+        elif backend.firmware.device == "flex":
+            navigator.navigate_until_text_and_compare(NavInsID.SWIPE_CENTER_TO_LEFT,
+                                                      [NavInsID.USE_CASE_REVIEW_CONFIRM, NavInsID.USE_CASE_STATUS_DISMISS],
+                                                      "Hold to sign",
+                                                      ROOT_SCREENSHOT_PATH,
+                                                      test_name)        
```

Comment:
`scenario_navigator.review `could be used here because it's a really standard use case (no need to provide instructions, same command for all devices)

Reply:
Done